arrayfire/image.py

```diff
@@ -711,6 +711,49 @@ def regions(image, conn = CONNECTIVITY.FOUR, out_type = Dtype.f32):
                                        conn.value, out_type.value))
     return output
 
+def confidenceCC(image, seedx, seedy, radius, multiplier, iters, segmented_value):
+    """
+    Find the confidence connected components in the image.
```

Comment:
Perhaps a link to our C++ documentation that shows an example set of images. You can add them here too but you would to copy those images to this repo again.

Reply:
The rest of our python documentation is picture-less. Avoiding those links for now.